Source/Settings.as

```diff
@@ -196,6 +208,47 @@ vec4 Setting_Gearbox_TextColor = vec4(1, 1, 1, 1);
 
 [Setting category="Gearbox" name="Font"]
 string Setting_Gearbox_Font = "DroidSans.ttf";
+
+
+[Setting category="Acceleration" name="Positive acceleration color" color]
+vec4 Setting_Acceleration_Positive_Color = vec4(0, 0.9f, 0, 1);
+
+[Setting category="Acceleration" name="Negative acceleration color" color]
+vec4 Setting_Acceleration_Negative_Color = vec4(0.8f, 0, 0, 1);
+
+[Setting category="Acceleration" name="Backdrop color" color]
+vec4 Setting_Acceleration_BackdropColor = vec4(0, 0, 0, 0.7f);
+
+[Setting category="Acceleration" name="Border color" color]
+vec4 Setting_Acceleration_BorderColor = vec4(1, 1, 1, 1);
+
+[Setting category="Acceleration" name="Border width" drag min=0 max=10]
+float Setting_Acceleration_BorderWidth = 3.0f;
+
+[Setting category="Acceleration" name="Border radius" drag min=0 max=50]
+float Setting_Acceleration_BorderRadius = 5.0f;
+
+[Setting category="Acceleration" name="Maximum acceleration value" drag min=0 max=200]
+float Setting_Acceleration_MaximumAcceleration = 15.0f;
+
+[Setting category="Acceleration" name="Show text value"]
+bool Setting_Acceleration_ShowTextValue = true;
+
+[Setting category="Acceleration" name="Text padding" drag min=0 max=100]
+float Setting_Acceleration_TextPadding = 20.0f;
+
+[Setting category="Acceleration" name="Padding between negative and position bars" drag min=0 max=100]
+float Setting_Acceleration_BarPadding = 7.5f;
+
+[Setting category="Acceleration" name="Text color" color]
+vec4 Setting_Acceleration_TextColor = vec4(1, 1, 1, 1);
+
+[Setting category="Acceleration" name="Font size" drag min=0 max=50]
+float Setting_Acceleration_FontSize = 15.0f;
+
+[Setting category="Acceleration" name="Enable smoothing"]
+bool Setting_Acceleration_Smoothing = true;
+
```

Comment:
There should be an option to change the font, just like the gearbox and all the other widgets.